src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+  dataUrl: "unity/Build/unity.data",
+  frameworkUrl: "unity/Build/unity.framework.js",
+  codeUrl: "unity/Build/unity.wasm",
+});
+
+
+function App() {
+  return (
+    <div className="App">
+    //insert the  unity webgl in your app
+       <Unity unityContext={unityContext} />
+    </div>
+  );
+}
+```
+Set a size for unity WebGL using css in ```src/index.css```
+```css
+canvas{
+  width: 840px;
+  height: 480px;
+}
+
+```
+### 4\. Configure UseDApp{#UseDApp-config}
+Let's use the UseDApp to get the account balance and send a transction,but first we need to configure it in ```src/index.tsx```
+```typescript
+import { ChainId, Config, DAppProvider } from '@usedapp/core';
+
+const config:Config  = {
+  //Choose your favorite network
+  readOnlyChainId: ChainId.Kovan,
+  readOnlyUrls: {
+    [ChainId.Kovan]: '<add a eth endpoint here>',
+  },
+}
+//Add UseDApp in your app
+ReactDOM.render(
+  <React.StrictMode>
+    <DAppProvider config={config}>
+      <App />
+    </DAppProvider>
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+```
+### 5\. UseDApp to communicate with unity{#UseDApp}
+Create the functions that will interact with unity, in ```sr/App.tsx``` let's write  the first part of code to interact with unity.
+```typescript
+  import { useEtherBalance, useEthers, useSendTransaction } from '@usedapp/core';
+  import { formatEther,formatUnits} from '@ethersproject/units'
+  
+  //This function will show the account balance in the unity. 
+  function EtherBalanceToUnity(amount : string) {
+    unityContext.send("/Canvas/Text", "EtherBalanceToUnity", amount);
+  }
+```
+Now modify the **App function** to show the account balance on unity and send a transaction when clicked in button inside the unity
+```typescript
+ const App = () => {
+   //Connect to wallet to access your account.
+  const { activateBrowserWallet, account } = useEthers()
+   //The function that will be used to send a transaction.
+  const { sendTransaction } = useSendTransaction()
+
+  //Get the balance in your account.
+  const etherBalance = useEtherBalance(account)
+  //Detect if the SendTranscationFromUnity method in unity is called,and open the wallet to send a new transaction.
+  unityContext.on("SendTranscationFromUnity",( address, amount) => {
+    sendTransaction({ to: address, value: amount })
+  });
+  return <div>
+    <div>
+     //Connect to your wallet
+      <button onClick={() => { 
+            activateBrowserWallet(); 
+          }}>Connect</button>
+          {account && <p>Account: {account}</p>}
+          //Show your account balance on unity
+          {etherBalance && EtherBalanceToUnity(formatEther(etherBalance))}
+    </div>
+    <div>
+      <Unity unityContext={unityContext} />
+    </div>
+  </div>
+    
+}
+```
+Run your app: ```npm run start```,
+
+When clicked in connect the UsaDApp will open your wallet and ask to connect to website,if you accept the connection, your account balance will be shown in the unity.
```

Comment:
Clicking "Connect" triggers UseDApp to open your wallet and ask to connect to the website. Accepting the connection allows your account balance to be shown within Unity.